### PR TITLE
feat(mongodb-runner): add CLI script MONGOSH-1488

### PR DESCRIPTION
**Pull request by addaleax**
#102 + one commit

### CODE REVIEW

File: packages/mongodb-runner/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# mongodb-runner
+
+Helper for spinning up MongoDB servers and clusters for testing.
+
+## Example usage
+
+> Note: Version 5 of mongodb-runner is a full re-write. Many things work
+> differently in previous versions.
```
Comment (lerouxb) [inline packages/mongodb-runner/README.md:8]: Total copywriting nitpick, totally just ignore me: Feels like normally you'd word this sentence that it works different _now_ not in the past.

File: packages/mongodb-runner/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  mongodb-runner start  Start a MongoDB instance
+  mongodb-runner stop   Stop a MongoDB instance
+  mongodb-runner ls     List currently running MongoDB instances
+  mongodb-runner exec   Run a process with a MongoDB instance (as MONGODB_URI
+                        env var)
+
+Options:
+  -t, --topology     Select a topology type
+           [choices: "standalone", "replset", "sharded"] [default: "standalone"]
+      --arbiters     number of arbiter nodes for each replica set       [number]
```
Comment (lerouxb) [inline packages/mongodb-runner/README.md:34]: Minor thing for all these numbers: Should we specify the default? Is there even a default in every case? I suppose each one only matters for specific code paths.

File: packages/mongodb-runner/src/cli.ts [hunk 3]
```diff
@@ -1,31 +1,203 @@
...
+      ...argv,
+      args: mongodArgs,
+    });
+    try {
+      const [prog, ...progArgs] = execArgs;
+      const child = spawn(prog, progArgs, {
+        stdio: 'inherit',
+        env: {
+          ...process.env,
+          // both spellings since otherwise I'd end up misspelling these half of the time
```
Comment (lerouxb) [inline packages/mongodb-runner/src/cli.ts:181]: 😄

**Review by lerouxb — APPROVED** [review]

File: packages/mongodb-runner/README.md (hunk 2 above)
Comment (addaleax) [reply packages/mongodb-runner/README.md:34]: So, just for context, the `default:` in yargs sets the actual default, it’s not just informative for the user … so setting e.g. `default: 2` for `secondaries` means that we’d maintain the default in two places. That might be a bit nicer for UX, but I’m not sure if it’s really worth it?

File: packages/mongodb-runner/README.md (hunk 1 above)
Comment (addaleax) [reply packages/mongodb-runner/README.md]: Sure, rephrased this a bit :+1:

File: packages/mongodb-runner/README.md (hunk 2 above)
Comment (lerouxb) [reply packages/mongodb-runner/README.md:34]: Yeah we're the only users, so probably not worth it.

**Pull request merged**